app/konnect/dev-portal/customization/index.md

```diff
@@ -27,16 +27,24 @@ Here, you have the ability to modify the following:
 
 For details on the requirements for each customizable option, hover over the information (`i`) icon next to each item in the UI.
 
-
 ## Custom Developer Portal URL
 
 All Dev Portals have an auto-generated default Dev Portal URL. You can add a custom domain. When set up properly, users can access the Dev Portal from both the default URL and the custom URL. The {{site.konnect_short_name}} Kong Developer Portal generates an SSL certificate for your custom domain automatically.
 
+
 ### Prerequisites
 
 * A domain and access to configure domain's CNAME
 * Your organization's auto-generated default Dev Portal URL. For example, `https://kong121212.portal.konnect.konghq.com/`.
 
+### Supported top-level domains
+
+|Supported top-level domains|
```

Comment:
```suggestion
```
You can leave this out to avoid having a heading that just duplicates the section name. It'll look like this:
![Screen Shot 2022-06-24 at 10 11 01 AM](https://user-images.githubusercontent.com/54370747/175609696-98733022-ce0c-42b4-9783-9307b3014892.png)

Alternatively, you can leave the table heading as is but move the table to just below the intro paragraph for "Custom Developer Portal URL"